install.sh

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2020, The Pallene Developers
+# Pallene is licensed under the MIT license.
+# Please refer to the LICENSE and AUTHORS files for details
+# SPDX-License-Identifier: MIT
+
+####
+# One script to rule them all, one script to find them,
+# one script to bring them all and in the ease-of-use bind them.
+####
+
+# Versions
+LUA_VERSION=5.4.7
+LUAROCKS_VERSION=3.9.1
+PT_VERSION=0.5.0a
+
+# Where we will clone
+CLONE_DIR="/tmp/pallene"
+mkdir -p $CLONE_DIR
+
+# Current working directory
+CURR_DIR=$(pwd)
+
+# Where to install/locate Lua
+LUA_PREFIX=${LUA_PREFIX:-/usr/local}
+
+# Function to display usage
+usage() {
+    echo "Usage: $0 { world | lua | rocks | ptracer | pallene }"
+    echo "  world    - Install all components"
+    echo "  lua      - Install Lua Internals"
+    echo "  pallene  - Install Pallene"
+    echo "  rocks    - Install LuaRocks"
+    echo "  ptracer  - Install Pallene Tracer"
+    echo
+    echo "Lua Internals:"
+    echo "  Use LUA_CFLAGS, LUA_LDFLAGS, LUA_LIBS and LUA_OBJS to pass flags:"
+    echo "    LUA_MYCFLAGS=-DLUAI_ASSERT ./install.sh lua; or"
+    echo "    LUA_CFLAGS=-fsanitize=address LUA_LDFLAGS=-lasan ./install.sh lua; to enable Address Sanitization"
+    echo
+    echo "  Use LUA_PLAT to define Lua platform. Defualt is 'linux'."
+    echo "  Available platforms: guess aix bsd c89 freebsd generic ios linux linux-readline macosx mingw posix solaris"
+    echo "    e.g. LUA_PLAT=linux-readline ./install.sh lua"
+    echo
+    echo "  Use LUA_PREFIX to provide Lua install prefix. Default is '/usr/local'."
+    echo "    e.g. LUA_PREFIX=/usr/local ./install.sh lua"
+    echo
+    echo "Pallene Tracer:"
+    echo "  Use PT_TESTS=1 to run Pallene Tracer tests after installation, e.g."
+    echo "    PT_TESTS=1 ./install.sh ptracer"
```

Comment:
Can we remove the PT_TESTS functionality? I would rather have the CI run those in a separate step than integrate them into the installation script. It complicates the script, and also introduces a dependency on busted.